Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/TransformToolIntegrationTest.java

```diff
@@ -703,6 +705,8 @@ public void testCropFromEverySideOnFilledBitmap() {
 				.performAutoCrop();
 		onDrawingSurfaceView()
 				.perform(touchAt(DrawingSurfaceLocationProvider.MIDDLE));
```

Comment:
This call is no longer needed since it is done with your performcheckmark function below. Please have a look if this is the case in any other tests too.